docs/user/SDarnRead.md

```diff
@@ -30,6 +30,22 @@ fitacf_data = SDarn_read.read_fitacf()
 ```
 where the named variable `fitacf_data` is a python dictionary list containing all the data in the file. If you were reading a different kind of file, you would need to use the methods `read_iqdat`, `read_rawacf`, `read_grid` or `read_map` for their respective filetypes.
 
+## Reading a compressed file
+
+To read a compressed file like **bz2** (commonly used for SuperDARN data products), you will need to use [bz2 library](https://docs.python.org/3/library/bz2.html). 
+The `SDarnRead` class allows the user to provide the file data as a stream of data which is what the **bz2** returns when it reads a compressed file: 
+```python
+import bz2
+import pydarn
+
+fitacf_file = path/to/file.bz2
+with bz2.open(fitacf_file) as fp:
+      fitacf_stream = fp.read()
+
+reader = pydarn.SDarnRead(fitacf_stream, True)
+records = reader.read_fitacf()
+```
+
```

Comment:
Okay just saw this here. Not sure if it's necessary in the readme as like I said, I don't know if this is a common way people handle files.

Reply:
Alex Chetier tried out pyDARN and right away tried to read a compressed file. Maybe others can chime in what is more preferred unzipping or reading in via compression. 
@kherli  @aburrell @asreimer

Reply:
I think the example should still exist in the documentation as we already had one person try it and email me why they cannot read a compressed file in. However, if it should be in the README is another question.

Reply:
Happy for it to exist in documentation. Need other opinions to decide the fate of the README